source/jsonnet/england-wales/ccs/blocks/who-lives-here/any_visitors.jsonnet

```diff
@@ -1,77 +1,99 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
+local rules = import 'rules.libsonnet';
+
+local householdAddressTitle = {
```

Comment:
Not a big fan of the duplication with effectively just one of the placeholder being different. I have been playing with passing the placeholder in but I am not sure its better, but it works. Its only rough and you could take it further by passing a dict

````
local question(placeholderAddress, placeholderAddressName) = {
  type: 'General',
  id: 'any-visitors-question',
  title: {
    text: 'How many visitors were staying overnight at '+ placeholderAddressName +' on Sunday {census_date}?',
    placeholders: [
        placeholderAddress,
        placeholders.censusDate,
    ],
  },
...
question_variants: [
    {
      question: question(placeholders.address, '{household_address}'),
      when: [rules.isHouseholdAddress],
    },
    {
      question: question(placeholders.otherAddress, '{other_address}'),
      when: [rules.isNotHouseholdAddress],
    },
  ]
```
Probably needs a 2nd opinion @MebinAbraham @ajmaddaford

Reply:
I think that the duplication is small and makes it much clearer.

However, the [style guide](https://github.com/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md#single-value-variants-example) should be followed i.e. the boolean value should be passed all the way through.

Reply:
I left that duplication untouched and added booleans all the way through.